core/FileHash.h

```diff
@@ -245,11 +249,10 @@ struct UsageHash {
 struct FileHash {
     LocalSymbolTableHashes localSymbolTableHashes;
     UsageHash usages;
-    FoundMethodHashes foundMethodHashes;
+    FoundHashes foundHashes;
```

Comment:
What do you think about `FoundDefHashes`?

Reply:
WFM.  I substituted the name, I did not change parameter names.  LMK if you'd like me to change this.

Reply:
lgtm